api/openapi.yaml.in

```diff
@@ -450,10 +450,14 @@ components:
             template parameters.
         name:
           type: string
-          description: Name of the SessionTemplate. The length of the name is restricted to 45 characters.
+          description: |
+            Name of the Session Template.
+            
+            It is recommended to use names which meet the following restrictions:
+            * Length of 1-45 characters.
+            * Use only lowercase letters, digits, periods (.), and dashes (-).
+            * Begin and end with a letter or digit.
           example: "cle-1.0.0"
-          minLength: 1
```

Comment:
By removing the 'minLength' designation will that allow the cli to pass a 0 length string in? How about the pattern - doesn't that enforce what you are describing above?

Reply:
None of those are currently enforced is the problem. This PR is changing the spec to reflect the actual behavior. Although I do suppose it does enforce the minimum length -- you are indeed not allowed to create session templates with 0-length names. So I can add that one back.